.github/ISSUE_TEMPLATE/use-case.md

```diff
@@ -0,0 +1,44 @@
+---
+name: Use Case
+about: Use this template to propose an LWS use case.
+title: "[UC] <<brief description of use case>>"
+labels: triage, usecase
+assignees: ''
+
+---
+
+**As a [primary actor],**
+**I want [an action or feature],**
+**So that [a reason or benefit].**
+
+## Preconditions:
+
+*What preconditions do we expect there to be fulfilled prior to this use case being triggered?*
+
+## Trigger:
+
+*What triggers this use case?*
```

Comment:
The term "trigger" is not particularly clear - as also mentioned in https://github.com/w3c/lws-ucs/pull/5#issuecomment-2444539553 . Perhaps what's intended is along the following lines?

```suggestion
*What (user or system) event or action initiates this use case?*
```